views/js/admin/order_add.js

```diff
@@ -32,32 +32,34 @@
  * @codingStandardsIgnoreStart
  */
 $(document).ready(function () {
-    var $paymentSelector = $('select[name="payment_module_name"]');
-    $paymentSelector.ready(function () {
-        $("#mollie-email-send-group").appendTo($paymentSelector.closest('div.form-group'));
-        isMollie = isMolliePayment($('select[name="payment_module_name"]').val());
+    var $paymentSelector = isPsVersion177 ? 'select[name="cart_summary[payment_module]"]' : 'select[name="payment_module_name"]';
+    var displayHiddenClass = 'd-none hidden';
+    var $paymentSelectInput = $($paymentSelector)
+
+    $paymentSelectInput.ready(function () {
+        $("#mollie-email-send-group").appendTo($paymentSelectInput.closest('div.form-group'));
+        isMollie = isMolliePayment($($paymentSelector).val());
         toggleOrderStatus(isMollie);
     });
 
-    $(document).on('change', 'select[name="payment_module_name"]', function () {
+    $(document).on('change', $paymentSelector, function () {
         var selectedPayment = $(this).val();
         var isMollie = isMolliePayment(selectedPayment);
-
         toggleOrderStatus(isMollie);
     });
 
     function toggleOrderStatus(isMolliePayment) {
         var $molliePaymentCheckboxGroup = $('#mollie-email-send-group');
-        var $orderStatusSelector = $('select[name="id_order_state"]');
+        var $orderStatusSelector = isPsVersion177 ? $('select[name="cart_summary[order_state]"]') : $('select[name="id_order_state"]');
```

Comment:
I think you can avoid conditions just by comma separating multiple values in jQuery.

Like this:

```javascript
$('select[name="cart_summary[order_state]"],select[name="id_order_state"]')
```
does this work ?

Reply:
Seems to work, changed it.